src/Items/ItemBoat.h

```diff
@@ -98,6 +98,12 @@ class cItemBoatHandler :
 		cBoat * Boat = new cBoat(x + 0.5, y + 0.5, z + 0.5);
 		Boat->Initialize(*a_World);
 
+		// Remove boat from players hand
+		if (!a_Player->IsGameModeCreative())
```

Comment:
Okay then, just asking (I have no idea how the Spectator mode works, haven't played Minecraft for ages, actually :)